homeassistant/components/verisure/__init__.py

```diff
@@ -103,6 +109,28 @@ def capture_smartcam(service):
         DOMAIN, SERVICE_CAPTURE_SMARTCAM, capture_smartcam, schema=CAPTURE_IMAGE_SCHEMA
     )
 
+    def disable_autolock(service):
```

Comment:
Please add decorator `@verify_domain_control(hass, DOMAIN)` on both service handlers to make sure the user is able to control at least a device in this domain.

You can import the decorator from `from homeassistant.helpers.ervice import verify_domain_control`

Reply:
I added that for the capture_smartcam service as well

Reply:
I don't get this to work. As I understand the owner should have permission to use every service but I get ```[homeassistant.components.websocket_api.http.connection.140539906835344] Unauthorized``` when i'm trying to call it. I have tried to remove my ```.homeassistant``` folder in order to recreate everything, but no luck. Am I missing something?

Reply:
@balloob , can you help me? Or would it be OK to wait with the permissions and add that in another PR?